dali/base/dasds.cpp

```diff
@@ -94,6 +95,11 @@ static unsigned msgCount=(unsigned)-1;
 // # TYPE sds_pending_requests gauge
 // A unsigned scalar should suffice, i.e. never going to have that many pending transactions
 
+static auto pSdsRequestsReceived = hpccMetrics::registerCounterMetric("dali.sds_requests.received", "The total number of Dali SDS requests received", SMeasureCount);
+static auto pSdsRequestsStarted = hpccMetrics::registerCounterMetric("dali.sds_requests.started", "The total number of Dali SDS requests started", SMeasureCount);
+static auto pSdsRequestsCompleted = hpccMetrics::registerCounterMetric("dali.sds_requests.completed", "The total number of Dali SDS requests completed", SMeasureCount);
+static auto pSdsRequestsPending = hpccMetrics::registerGaugeFromCountersMetric("dali.sds_requests.pending", "Current number of pending SDS requests", SMeasureCount, pSdsRequestsReceived, pSdsRequestsStarted);
```

Comment:
did we drop the idea of tracking pending request?
I think that makes sense, because we know # received, number started and number completed so can calculate number blocked pending.
But if so, this line should be deleted.

Reply:
@jakesmith this line automatically calculates and publishes the number.

Reply:
right ok, yes I'd missed what it was doing.